#166272289 Chore Set up redux

despeauxz opened the pull request:
#### What does this PR do?
This PR sets up Redux into the application
#### Description of Task to be completed?
- Set up Redux to the application
#### How should this be manually tested?
- clone the repo
- install the packages via `npm install`
- start the dev-server with `npm run start:dev`
- install `Redux chrome devtool` as an extension on chrome
- Open your console and check for the `Redux` tab
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166272289](https://www.pivotaltracker.com/story/show/166272289)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

despeauxz committed:
ch-[166272289]: setup redux

- setup the provider from react-redux
- configured the store
- initialized an empty reducer

[#166272289]

despeauxz committed:
ch-[166272289]: setup redux

- setup the provider from react-redux
- configure the store
- initialize an emoty reducer
- setup PR template

[#166272289]

fidelisojeah commented on src/components/App/index.jsx:
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+
+const App = () => (
```
I remember mentioning you guys should not create your components like this this morning

```suggestion
function App() {
  return (
```

despeauxz replied:
I'll fix it right now, thanks

despeauxz committed:
ch-[166272289]: setup redux

- setup the provider from react-redux
- configure the store
- initialize an emoty reducer
- setup PR template

[#166272289]

adex001 reviewed (dismissed):
Looks good to me.

despeauxz committed:
ch-[166272289]: setup redux

- setup the provider from react-redux
- configure the store
- initialize an emoty reducer
- setup PR template

[#166272289]

despeauxz committed:
ch-[166272289]: setup redux

- setup the provider from react-redux
- configure the store
- initialize an emoty reducer
- setup PR template

[#166272289]

despeauxz committed:
chore-[166272289]: setup redux

- setup the provider from react-redux
- configure the store
- initialize an emoty reducer
- setup PR template

[#166272289]

despeauxz committed:
Merge branch 'develop' into 166272289-ch-setup-redux

despeauxz committed:
chore-[166272289]: setup redux

- setup the provider from react-redux
- configure the store
- initialize an emoty reducer
- setup PR template

[#166272289]

despeauxz committed:
chore-[166272289]: setup redux

- setup the provider from react-redux
- configure the store
- initialize an emoty reducer
- setup PR template

[#166272289]

adex001 reviewed (dismissed):
Looks good. Works great! keep it up

despeauxz committed:
chore-[166272289]: setup redux

- setup the provider from react-redux
- configure the store
- initialize an emoty reducer
- setup PR template

[#166272289]

despeauxz committed:
chore-[166272289]: setup redux

- setup the provider from react-redux
- configure the store
- initialize an emoty reducer
- setup PR template

[#166272289]

despeauxz committed:
chore-[166272289]: setup redux

- setup the provider from react-redux
- configure the store
- initialize an emoty reducer
- setup PR template

[#166272289]

EMacco reviewed (approved):
Good job
